### PR TITLE
Creates PDF / ZIP on user's choice

### LINKED ISSUE #66: Create PDF / ZIP only if the User asks for it
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/badgeyay

**Current behavior:**

**Expected behavior:**

**Steps to reproduce:**

Right now if a user uploads a CSV - we are making SVGs, PDFs and ZIP - and then giving the user the option to download one. This will waste time for large number of badges.

Maybe ask the user before-hand what all formats they want the badges in,

### PR DESCRIPTION
**Pull request by gabru-md**
Closes #66 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

- [X] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
This gives user the choice to get either PDFs or ZIPs or both as output results.
Screenshots  : 
screenshot at 2017-09-16 14-07-14
screenshot at 2017-09-16 14-07-42
screenshot at 2017-09-16 14-07-51

#### Changes proposed in this pull request:

- updated `index.html`
- changed `main.py`
- tweaked `merge_badges.py` from #71 

Note: Please merge this after #71. Because I will have to resolve some conflicts when PR71 gets merged.
Thanks!

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -118,4 +130,4 @@ def Not_Found(e):
 
 
 if __name__ == '__main__':
-	app.run()
+	app.run(debug=True)
```
Comment (jajodiaraghav) [inline app/main.py]: Why do we need `debug=True` here?

**Comment by jajodiaraghav** [comment]
Hi @gabru-md 
Great work! Made a comment. Rest looks fine :)

What do you think @niccokunzmann ?

File: app/main.py [hunk 2]
```diff
@@ -51,6 +56,9 @@ def upload():
 	csv = request.form['csv'].strip()
 	img = request.form['img-default']
 	file = request.files['file']
+	_pdf = True if request.form.get('pdf') == 'on' else False
+	_zip = True if request.form.get('zip') == 'on' else False
+	print(_zip,_pdf)
```
Comment (jajodiaraghav) [inline app/main.py]: Do we really need this print here?

File: app/templates/index.html [hunk 3]
```diff
@@ -13,14 +13,36 @@
...
 				{% endif %}
 			{% endwith %}
 			<form action="{{ url_for('upload') }}" method="post" enctype="multipart/form-data">
 			    <div class="form-group">
+			    	<label>Tick the download type(s):</label>
+			    	<label for="zip">Zip</label>
+			    	<input type="checkbox" id="zip" name="zip">
+			    	<label for="pdf">PDF</label>
+			    	<input type="checkbox" id="pdf" name="pdf">
+			    	<br>
```
Comment (jajodiaraghav) [inline app/templates/index.html]: Please replace this `<br>` and use css margin to get space here.

Comment (gabru-md) [reply app/templates/index.html]: can you help me out with this one?

**Comment by gabru-md** [comment]
I forgot to remove these commands before pushing them into my repository!
Just cleaning it ! :+1:

File: app/main.py (hunk 1 above)
Comment (gabru-md) [reply app/main.py]: removing it as well :+1:

File: app/templates/index.html (hunk 3 above)
Comment (jajodiaraghav) [reply app/templates/index.html]: Sure! 
Step 1: Remove "br" tag
Step 2: Wrap line 40 to 44 inside <span class="options">
Step 3: Add this to `style.css`
```
.options{
margin-bottom: 20px;
}
```

Comment (gabru-md) [reply app/templates/index.html]: i'm supposed to wrap it inside what ?

Comment (jajodiaraghav) [reply app/templates/index.html]: @gabru-md Sorry, forgot to add "`"

**Comment by gabru-md** [comment]
@jajodiaraghav Please review now.
I've made the necessary changes.
:D

**Comment by jajodiaraghav** [comment]
Everything looks good! Now please squash your commits. You have made 6 commits for just one task.
Squash them.
Step 1: `git rebase -i HEAD~6`. An editor opens.
Step 2: Replace pick with squash for second to sixth line. Save and close.
Step 3: `git push origin gabru-md/dev -f`

**Comment by gabru-md** [comment]
It seems that I've messed things up. Can you help me ?

**Comment by jajodiaraghav** [comment]
To fix:

Step 1: `git rebase -i HEAD~3`. An editor opens.
Step 2: Replace pick with drop for "Add documentation of heroku deployment (#60)" and "Add option to select from default backgrounds (#68)" line. Save and close.
Step 3: `git push origin gabru-md/dev -f`

**Comment by gabru-md** [comment]
@jajodiaraghav please review

**Comment by S2606** [comment]
@jajodiaraghav should i merge this??

**Comment by jajodiaraghav** [comment]
@gabru-md You did really well. Although the purpose of rebasing wasn't achieved as you still pushed 2 commits.
Just some advice for the future contibutions:
1. When pulling from upstream branch, ALWAYS use `git pull upstream master<development in our case> --rebase`. 
Notice `--rebase` in the line

2. If you have already made a commit for a task, use `git commit --amend --no-edit` instead of `git commit -m "message"`. Google this.

Merging 🏎

**Comment by gabru-md** [comment]
@jajodiaraghav I would like to connect to you for future assistance in this matter.
PS : I have sent you a request on facebook.

**Pull request merged**